examples/generalized_linear_models/GLM-robust.ipynb

```diff
@@ -16,7 +16,7 @@
     "\n",
```

Comment:
<p>With Bambi 0.6.0, it's not necessary to use a custom<code>Family</code>  anymore. You can just call <code>family="t"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>